packages/web-pkg/src/components/FilesList/ResourceTable.vue

```diff
@@ -172,6 +172,7 @@
       <oc-icon
         v-oc-tooltip="item.disabled ? $gettext('Disabled') : $gettext('Enabled')"
         :name="item.disabled ? 'stop-circle' : 'play-circle'"
+        :accessible-label="`${$pgettext('use by reader to announce spaces\'s  status', 'Space is')} ${item.disabled ? $gettext('Disabled') : $gettext('Enabled')}`"
```

Comment:
Use [interpolation](https://jshmrtn.github.io/vue3-gettext/functions.html#interpolation-1) instead as some languages might use different format, order, etc.

Also, is this really about disabled space? I would like to play this safe and would suggest double checking whether a resource can be disabled while it's parent space is enabled...